tests/src/Context/ScreenshotContext.php

```diff
@@ -26,7 +64,7 @@ class ScreenshotContext extends RawMinkContext {
    * @Then (I )take a screenshot :name
    */
   public function takeScreenshot($name = NULL) {
-    $file_name = sys_get_temp_dir() . DIRECTORY_SEPARATOR . $name;
+    $file_name = $this->screenshotsDir . DIRECTORY_SEPARATOR . $name;
```

Comment:
Since `$this->screenshotsDir` is being checked in `::createScreenshot()` to see if it is valid we should not rely on it here.